core/object/make_virtuals.py

```diff
@@ -47,8 +47,8 @@
 }\\
 _FORCE_INLINE_ bool _gdvirtual_##m_name##_overridden() const { \\
 	ScriptInstance *_script_instance = ((Object*)(this))->get_script_instance();\\
-	if (_script_instance) {\\
-	    return _script_instance->has_method(_gdvirtual_##m_name##_sn);\\
+	if (_script_instance && _script_instance->has_method(_gdvirtual_##m_name##_sn)) {\\
+        return true;\\
```

Comment:
Ah, good catch! This change does seem correct: we shouldn't stop at the script for every method.

Reply:
Verified that I could have both kind of methods on an object with just this change.